src/devices/Hmc5883l/Status.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Hmc5883l
+{
+    /// <summary>
+    /// The status of HMC5883L device
+    /// </summary>
+    public enum  Status
+    {
+        /// <summary>
+        /// Regulator Enabled Bit. This bit is set when the internal voltage regulator is enabled.
+        /// This bit is cleared when the internal regulator is disabled.
+        /// </summary>
+        Ren,
```

Comment:
I'd call it full names if possible so just RegulatorEnabled

Reply:
Fixed